charts/shoot-core/components/charts/kube-proxy/templates/kube-proxy-init-script.yaml

```
@@ -0,0 +1,25 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kube-proxy-init-script
```

```suggestion
  name: kube-proxy-cleanup-script
```

Please, add some labels, too.
```yaml
    app: kubernetes
    gardener.cloud/role: system-component
    origin: gardener
    role: proxy
```